topics/basics/binding_parameters/README.md

```diff
@@ -13,7 +13,7 @@ cass_statement_bind_string(statement, 0, "abc");
 cass_statement_free(statement);
 ```
 
-Variables can only be bound by name for prepared statements. This limitation exists because query metadata provided by Cassandra is required to map the variable name to the variable’s marker index.
+Variables can only be bound by name for [prepared statements](../prepared_statements/). This limitation exists because query metadata provided by Cassandra is required to map the variable name to the variable’s marker index. Named bind variables are specified as `:variable` inside query string.
```

Comment:
This is no longer a limitation. This was just out-of-date. Simple statements now support named `?` parameters since the protocol v3 (or Cassandra 2.1).

So Cassandra 1.2/2.0 have this limitation and Cassandra 2.1+ does not.

Reply:
Corrected it